ingestion/src/metadata/ingestion/source/pipeline/airflow/metadata.py

```diff
@@ -266,7 +266,9 @@ def yield_pipeline_status(
                         executionStatus=STATUS_MAP.get(
                             dag_run.state, StatusType.Pending.value
                         ),
-                        timestamp=Timestamp(timestamp) if timestamp else None,
+                        timestamp=Timestamp(
+                            timestamp or datetime_to_ts(datetime.now())
```

Comment:
if timestamp is None I don't think we should ingest that status. Basically we would be giving out false alerts of failed status that maybe occured weeks ago.